src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -130,6 +130,11 @@ export default ({
     [property?.default?.bgcolor],
   );
 
+  // scene mode
```

Comment:
I don't think we need this comment since the code is pretty self-explanatory

Reply:
updated